lnbits/templates/base.html

```diff
@@ -34,10 +34,12 @@
           {% endblock %}
           <q-toolbar-title>
             <q-btn flat no-caps dense size="lg" type="a" href="/">
-              {% block toolbar_title %} {% if SITE_TITLE != 'LNbits' %} {{
-              SITE_TITLE }} {% else %} <strong>LN</strong>bits {% endif %} {%
-              endblock %}</q-btn
-            >
+              {% block toolbar_title %} {% if USE_CUSTOM_LOGO %}
+              <img height="30px" alt="Logo" src="{{ USE_CUSTOM_LOGO }}" />
+              {%else%} {% if SITE_TITLE != 'LNbits' %} {{ SITE_TITLE }} {% else
+              %}
+              <strong>LN</strong>bits {% endif %} {%endif%} {% endblock %}
+            </q-btn>
```

Comment:
I read: "if there is a custom logo, use it, otherwise set the page title" shouldn't the page title always be set?